components/automate-ui/src/app/types/types.ts

```diff
@@ -818,7 +818,21 @@ export enum HttpStatus {
   FORBIDDEN = 403,
   NOT_FOUND = 404,
   CONFLICT = 409,
-  PRECONDITION_FAILED = 412,
+  PRECONDITION_FAILED = 412, // Warning! Grpc.FailedPrecondition maps to 400, not 412!
```

Comment:
🤔 ooh just saw this. can we make our API return the correct HTTP code?

Reply:
Nope. That's what gRPC does.